build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,331 @@
+apiVersion: v1
```

Comment:
Please also add parameter in https://github.com/antrea-io/antrea/blob/1df3ac4771d5e761cf22863fab685048afae3381/hack/generate-manifest-windows.sh#L61 to support new yaml generation.

Reply:
done, PTAL.
Thanks